Wikipedia/Code/VanishAccountContentView.swift

```diff
@@ -0,0 +1,196 @@
+import SwiftUI
+import WMF
+
+struct VanishAccountContentView: View {
+    
+    enum LocalizedStrings {
+        static let title = WMFLocalizedString("vanish-account-title", value: "Vanishing process", comment: "Title for the vanishing process screen")
+        static let description = WMFLocalizedString("vanish-account-description", value: "Vanishing is a last resort and should only be used when you wish to stop editing forever and also to hide as many of your past associations as possible.\n\nTo initiate the vanishing process please provide the following", comment: "Description for the vanishing process")
+        static let usernameFieldTitle = WMFLocalizedString("vanish-account-username-field", value: "Username and user page", comment: "Title for the username and userpage form field")
+        static let additionalInformationFieldTitle = WMFLocalizedString("vanish-account-additional-information-field", value: "Additional information", comment: "Title for the additional information form field")
+        static let additionalInformationFieldPlaceholder = WMFLocalizedString("vanish-account-additional-information-placeholder", value: "Optional", comment: "Placeholder for the additional information form field")
+       
+        static let buttonText = WMFLocalizedString("vanish-account-button-text", value: "Send request", comment: "Text for button on vanish account request screen")
+    }
+    
+    @SwiftUI.ObservedObject var userInput: UserInput
+    @SwiftUI.State var toggleModalVisibility = false
```

Comment:
@mazevedofs  - my mistake, I can see how two different ones are needed now. Do you mind renaming this one to `isModalVisible`? That might help me read the distinction.

Reply:
When calling `openMailClient` I set `shouldShowModal` to true; if I merge them, it shows the modal immediately when pressing the button, and since it toggles `toggleModalVisibility ` when the app is foregrounded again, the modal appears for a moment and disappears, that's why I kept it separated.